nostr-client-plus/src/bin/aggregator.rs

```diff
@@ -54,6 +54,8 @@ async fn run() -> Result<()> {
     // Command line parsing
     let relay_url = std::env::var("RELAY_URL").unwrap_or("ws://127.0.0.1:3033".to_string());
     let supported_version = std::env::var("VERSION").unwrap_or("v0.0.1".to_string());
+    // Anything except empty, will be considered as true
+    let skip_pow_check = std::env::var("SKIP_POW_CHECK").unwrap_or("".to_string());
```

Comment:
is it the common practice? normally unless it's set as "true" specifically it's always false. it will be weird if it's set as "false" but take it as true

Reply:
yes, I think can compare the env to string "true"

Reply:
OK let's fix this and then merge